### ISSUE TITLE
Saving file output correctly

### ISSUE THREAD

**Issue by downthecrop** [chrippa/livestreamer#625]
Hey I was wondering how I'm am supposed to use the -o command correctly. I get the file output but that file seems have issues. The file is always encodes as h264 MPG-4 part 10. What file extention should I be using to save these files? .mp4? Any web based video player (tried all the common jquery players and HTML 5 video tags) can't load the file, I read in another issue thread that the file could be missing metadata (seeking information)? The output files load fine in VLC and windows media player regardless of extention once downloaded locally.

Tldr:  How can I "fix" a video recoded from an HLS twitch stream to be able to be read by a web based video player (flowplayer, jPlayer or HTML 5 video embed) and what file extention should I be using for these files?

**Comment by campones** [comment]
I can't insert the "  |  " in win shell.. any ideas?
